### PR TITLE
Some very simple features

### PR DESCRIPTION
**Pull request by yasmasdas**
Some very simples features
- Disable Phone
- Request Ballistic equipment and Request Boat Pickup
- Keep Vehicle On Ground

### CODE REVIEW

File: src/backend/looped/self/disable_phone.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#include "natives.hpp"
+#include "backend/looped_command.hpp"
+#include "script_global.hpp"
+
+namespace big
+{
+	class keep_on_ground : looped_command
```
Comment (Yimura) [inline src/backend/looped/self/disable_phone.cpp]: command is called keep on ground same for the global `g_keep_on_ground`

**Review by Yimura — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Disable phone already exists? What about being able to use the phone but disabling incoming calls?

**Comment by Aure7138** [comment]
I suggest not putting the globals in the command files, but putting it in the util or in one place for easy updating

**Comment by yasmasdas** [comment]
> Disable phone already exists? What about being able to use the phone but disabling incoming calls?

You're right I did not see that and I'll put the globals in a diff file later today i'll draft this pr

**Pull request closed without merging**